### DISCUSSION TITLE
Is it possible to pass successful Deezer downloads to a Windows installed Lidarr like with the old AMD Script?

### DISCUSSION THREAD

**Discussion by Guy767** [RandomNinjaAtk/arr-scripts#245] [Q&A]
Hello. I have lscr.io Lidarr installed in a Windows WSL Docker container running well. But I just want the download functionality of the arr script and not have an additional installation of Lidarr with a separate database and music library on my Windows PC just to procure missing albums from Deezer.

With the old AMD Docker script I was able to pass all successful Deezer downloads by specifying my Lidarr App key in AMD’s settings and remote path mapping Windows Lidarr’s Usenet Blackhole to AMD’s finished download location.

Is it it possible for the arr-script to not have the lscr.io Lidarr process completed downloads but have my Windows Lidarr do it instead? I would use the old AMD script but it doesn’t seem to work anymore. (Can not get data from Musicbrainz)

**Comment by RandomNinjaAtk** [comment]
Are you trying to migrate off of using a container? You seem to indicate that your already using the container and you could just add it to it....

On another note, no it is not designed to be used outside of the container, you could probably modify it to get it do that, but you'd still have to run the scripts on a linux system.

**Comment by Guy767** [comment]
Thank you for the reply. I already have Lidarr installed for Windows directly without a container. I'm only using the lscr.io version with the arr script so I can get Deezer to automatically fill missing albums by using an imported list from my main Windows Lidarr installation.

I really don't like having Docker containers running 24/7 on my system and I prefer to only run them when needed. About once a month I would try to fill my missing music that Usenet indexers failed to auto find with the AMD docker container. But since AMD is no longer working I'm forced to run an additional Lidarr installation complete with its own superfluous database and music library just to get the arr/AMD script functionality.

I was wondering if you could add a configuration setting in the Lidarr arr script where you could specify an address and app key for a separate Lidarr installation? Basically make it work like the old AMD script/container if wanted/needed.

**Reply by RandomNinjaAtk** [reply]
I will gladly review a PR that can implement the required changes, as long as it doesn't break existing functionality. But I do not have the cycles to design/implement a change that won't be of use to myself.

**Reply by airjer** [reply]
Was this implemented? I run all of my stuff in Windows.

**Reply by Guy767** [reply]
> Was this implemented? I run all of my stuff in Windows.

I did find a work around by using the Deemix plugin on the "Plugins" branch of Lidarr for Windows.

Via the Lidarr on Steroids Docker container I'm able to have Deemix as my Download client and Indexer for my Windows Installed Lidarr. (Need to specify the address "http://localhost:6595/"). Unfortunately running Deemix on its own via Windows or Docker container doesn't work for me; only the Lidarr on Steroids Deemix version does.

These somewhat convoluted steps may seem superfluous/unnecessary but I don't like having docker containers running 24/7 on my Windows PC and only run them when needed. Usually my Windows Lidarr finds most of my missing media on it's own via Usenet indexers but about 2 months or so I do like to search alternate sources for missing fills.

The good thing about the old AMD script is that it was able run without an attached/required Lidarr install; you could just point to an alternate Lidarr location. I'm sure the current arr script has a specified "hidden" Lidarr address and Api Key within its configuration files but it's beyond me to find/change it.

It would be great if the arr script had the option of changing the Lidarr address location and Api key; just like the old AMD script which doesn't work anymore unfortunately (Can not get data from Musicbrainz)...
